settings.cmake

```
@@ -32,9 +32,9 @@ endif()
 if(${TUT_BOARD} STREQUAL "pc")
     set(KernelArch "x86" CACHE STRING "" FORCE)
     set(KernelPlatform "pc99" CACHE STRING "" FORCE)
-    if(${TUT_ARCH} STREQUAL "ia32")
+    if("${TUT_ARCH}" STREQUAL "ia32")
         set(KernelSel4Arch "ia32" CACHE STRING "" FORCE)
-    elseif(${TUT_ARCH} STREQUAL "x86_64")
+    elseif("${TUT_ARCH}" STREQUAL "x86_64")
         set(KernelSel4Arch "x86_64" CACHE STRING "" FORCE)
     else()
         message(FATAL_ERROR "Unsupported PC architecture ${TUT_ARCH}")
```

I assume you have been running in an issue because the variabel was empty or not set. Maybe we should add quotes here to this message looks a bit nicer if this variable is empty.
```suggestion
        message(FATAL_ERROR "Unsupported PC architecture '${TUT_ARCH}'")
```

---

This is fixed in #89. Thanks.